integration-tests/tests/singlestore/test/init.sql

```diff
@@ -0,0 +1 @@
+CREATE database drizzle;
```

Comment:
what is this doing? I noticed the MySQL integration tests dir does not have this, is it required and did I just miss something?